src/Filter/Uglifyjs.php

```diff
@@ -41,8 +53,14 @@ class Uglifyjs extends AssetFilter
      */
     public function output($filename, $input)
     {
-        $cmd = $this->_settings['node'] . ' ' . $this->_settings['uglify'] . ' - ' . $this->_settings['options'];
+        $files = implode(' ', $this->files);
+        $cmd =
+            $this->_settings['node'] . ' ' .
+            $this->_settings['uglify'] . ' ' .
+            $files . ' ' .
+            $this->_settings['options'];
+
         $env = array('NODE_PATH' => $this->_settings['node_path']);
-        return $this->_runCmd($cmd, $input, $env);
+        return $this->_runCmd($cmd, '', $env);
```

Comment:
Won't this break filter chains that involve pre-processors and uglify? For example combining the coffescript or sprockets filter and uglify will no longer work as uglify will be told to use the source files and not the transformed results of those files.